### PR TITLE
Update role ocp4-workload-iot-managed-user for num of supported users

### PR DESCRIPTION
**Pull request by hguerrero**
##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

iot managed 

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: ansible/roles/ocp4-workload-iot-managed-user/templates/che-devfile.yaml.j2 [hunk 1]
```diff
@@ -19,9 +19,26 @@ components:
...
+    mountSources: true
+    memoryLimit: 4096M
+    type: dockerimage
+    volumes:
+      - name: m2
+        containerPath: /home/user/.m2
+    image: 'quay.io/eclipse/che-quarkus@sha256:0bd071932235090e0a260e0fea084d6fc82f74ecad46d1040e5026d6a627e72a'
+    alias: 'quarkus-nodejs-tools'
+    env:
+      - value: '-XX:MaxRAMPercentage=50.0 -XX:+UseParallelGC -XX:MinHeapFreeRatio=10 -XX:MaxHeapFreeRatio=20 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90 -Dsun.zip.disableMemoryMapping=true -Xms20m -Djava.security.egd=file:/dev/./urandom -Duser.home=/home/user'
```
Comment (jkupferer) [inline ansible/roles/ocp4-workload-iot-managed-user/templates/che-devfile.yaml.j2:35]: Please fix the long line issue here, recommend:
```
      - name: JAVA_OPTS
        value: >-
          -XX:MaxRAMPercentage=50.0
          -XX:+UseParallelGC
          -XX:MinHeapFreeRatio=10
          -XX:MaxHeapFreeRatio=20
          -XX:GCTimeRatio=4
          -XX:AdaptiveSizePolicyWeight=90
          -Dsun.zip.disableMemoryMapping=true
          -Xms20m
          -Djava.security.egd=file:/dev/./urandom
          -Duser.home=/home/user
```

File: ansible/roles/ocp4-workload-iot-managed/tasks/provision_camelk.yaml [hunk 2]
```diff
@@ -16,4 +16,31 @@
...
+    kind: ClusterServiceVersion
+    name: "{{ r_subscription_camelk.resources[0].status.currentCSV }}"
+    namespace: openshift-operators
+  register: r_csv_camelk
+  retries: 30
+  delay: 5
+  until:
+  - r_csv_camelk.resources[0].status.phase is defined
+  - r_csv_camelk.resources[0].status.phase | length > 0
+  - r_csv_camelk.resources[0].status.phase == "Succeeded"
```
Comment (jkupferer) [inline ansible/roles/ocp4-workload-iot-managed/tasks/provision_camelk.yaml:46]: Missing newline at end of file.

File: ansible/roles/ocp4-workload-iot-managed/templates/che-devfile.yaml.j2 [hunk 3]
```diff
@@ -19,9 +19,26 @@ components:
...
+    mountSources: true
+    memoryLimit: 4927M
+    type: dockerimage
+    volumes:
+      - name: m2
+        containerPath: /home/user/.m2
+    image: 'quay.io/eclipse/che-quarkus@sha256:0bd071932235090e0a260e0fea084d6fc82f74ecad46d1040e5026d6a627e72a'
+    alias: 'quarkus-nodejs-tools'
+    env:
+      - value: '-XX:MaxRAMPercentage=50.0 -XX:+UseParallelGC -XX:MinHeapFreeRatio=10 -XX:MaxHeapFreeRatio=20 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90 -Dsun.zip.disableMemoryMapping=true -Xms20m -Djava.security.egd=file:/dev/./urandom -Duser.home=/home/user'
```
Comment (jkupferer) [inline ansible/roles/ocp4-workload-iot-managed/templates/che-devfile.yaml.j2:35]: Another long line issue.

**Review by jkupferer — CHANGES_REQUESTED** [review]

**Comment by jkupferer** [comment]
Looks like just a few yamllint issues. Recommend checking with `yamllint` locally.

**Comment by hguerrero** [comment]
addressed requests in  https://github.com/redhat-cop/agnosticd/pull/3026/commits/3455d495da168e6574f40f756eb0200e58e554bd

**Review by jkupferer — APPROVED** [review]

**Pull request merged**